packages/axes/rollup.config.js

```diff
@@ -1,8 +1,8 @@
 const buildHelper = require("@egjs/build-helper");
 
 const external = {
-	"@egjs/agent": "eg.agent",
-	"@egjs/component": "eg.Component",
+  "@egjs/agent": "eg.agent",
+  "@egjs/component": "eg.Component",
```

Comment:
@egjs/component is registered in global as `Component` after v3.
- https://github.com/naver/egjs-component/blob/master/rollup.config.js

You should fix this to:
```
"@egjs/component": "Component"
```